python/lsst/dax/distribution/DataIngest.py

```diff
@@ -0,0 +1,297 @@
+    json formated requests and responses.
+
+    Parameters
+    ----------
+    host : str
+        Data ingest server host.
+    port : int
+        Data ingerst server port number.
+    user, auth_key : str, optional
+        User name and authorization key.
+    """
+
+    def __init__(self, host, port, user='', auth_key=''):
+        self._host = host
+        self._port = port
+        self._user = user
+        self._auth_key = auth_key
+        self._base_url = 'http://' + self._host + ':' + str(port) + '/'
+        print("base_url=", self._base_url)
+
+    def __repr__(self):
+        out = ("host=" + self._host + ":" + str(self._port)
+             + " user=" + self._user + "auth_key:****")
+        return out
+
+    def isIngestAlive(self):
+        """Test if ingest system is alive.
+        """
+        url = self._base_url + 'meta/version'
+        print("ingest url=", url)
+        response = requests.get(url)
+        r_json = response.json()
+        if not r_json['success']:
+            print('error: ' + r_json['error'])
+            return False
+        print('ingest version=', r_json['version'])
+        return True
+
+    def _postToIngest(self, ingest_cmd, data_json):
+        """Send the data_json to the ingest system and return
+        a json result
+        """
+        url = self._base_url + ingest_cmd
+        print('url=', url, " data=", data_json)
+        response = requests.post(url, json=data_json)
+        r_json = response.json()
+        success = True
+        if not r_json['success']:
+            print('ERROR post url=', url, " data=", data_json,
+                  'error: ', r_json['error'])
+            success = False
+        return success, r_json
+
+    def _putToIngest(self, ingest_cmd, data_json):
+        """Send the data_json to the ingest system and return
+        Return
+        ------
+        success : bool
+            True indicates operation was successful
+        status_code : int
+            Status code value from put request.
+        r_json : json or None
+            If the status code was not 200, this will be None.
+            Otherwise it is a json object with information about the request.
+        """
+        url = self._base_url + ingest_cmd
+        print('url=', url, " data=", data_json)
+        response = requests.put(url, json=data_json)
+        status_code = response.status_code
+        success = True
+        # 200 means the put request was at least well formed.
+        if status_code != 200:
+            print('ERROR put url=', url, "data=", data_json, 'status=', status_code)
+            success = False
+            return success, status_code, None
+        r_json = response.json()
+        if not r_json['success']:
+            print('ERROR put url=', url, 'status=', status_code, 'data=', data_json,
+                  'r_json=', r_json)
+            success = False
+        return success, status_code, r_json
+
+    def sendDatabase(self, db_file_path):
```

Comment:
Yes, indeed, this method is sending some JSON object to the corresponding service. However, this is not this operation is meant to do. The right name for the method should be `registerDatabase` or `createDatabase`.